.github/workflows/preview-links.yml

```diff
@@ -0,0 +1,16 @@
+name: Read the Docs Pull Request Preview
+on:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
@grugnog I didn't see this workflow run. Not sure why it didn't.

Reply:
This is fixed now